src/map/status.cpp

```diff
@@ -13859,6 +13879,18 @@ TIMER_FUNC(status_change_timer){
 		}
 		break;
 
+	case SC_GRADUAL_GRAVITY:
+		if (--(sce->val4) >= 0) {
+			int hp = status->max_hp * sce->val2 / 100;
+
+			if (!status_charge(bl, hp, 0))
+				status_zap(bl, hp, 0);
+			if (sc->data[type])
+				sc_timer_next(1000 + tick);
+			return 0;
```

Comment:
Should be rewritten to use status_get_sc_interval. Check SC_SHIELDSPELL_HP for reference.

Reply:
Yes but I guess we need to add `if (sc->data[type])` on using `status_get_sc_interval` part because without it server will print error when player die during duration of `SC_GRADUAL_GRAVITY`.